src/components/sbb-footer/sbb-footer.tsx

```diff
@@ -7,67 +7,33 @@ import { InterfaceFooterAttributes } from './sbb-footer.custom';
   tag: 'sbb-footer',
 })
 export class SbbFooter {
-  /** Footer appearance */
-  @Prop() public appearance?: InterfaceFooterAttributes['appearance'] = 'primary';
+  /**
+   * Negative coloring variant flag
+   */
+  @Prop() public negative: boolean;
+
+  @Prop() public variant?: InterfaceFooterAttributes['variant'] = 'default';
```

Comment:
Since the default value is set, maybe the optional can be removed?
```suggestion
  @Prop() public variant: InterfaceFooterAttributes['variant'] = 'default';
```